Components/Slider/slider.css

```diff
@@ -0,0 +1,34 @@
+.slider-container {
+  width: 10rem;
+}
+
+.slider {
+  appearance: none;
+  width: 100%;
+  height: 0.4rem;
+  border-radius: 0%;
+  background: lightgray;
+  outline: none;
```

Comment:
Better to use hex code instead of colour names.

Reply:
Thanks for the suggestions ! Just wanted to know for information, why hex codes are better than colour names ?

Reply:
Not sure but I think hex codes are universally supported in all browsers and these color names came up recently so some browser may have issues interpreting these name.

Reply:
Slider background now uses a color variable